tests/unit/nupic/encoders/date_test.py

```diff
@@ -34,17 +34,17 @@
 
 
 
-#########################################################################
 class DateEncoderTest(unittest.TestCase):
-  '''Unit tests for DateEncoder class'''
+  """Unit tests for DateEncoder class"""
 
 
   def setUp(self):
-    ##TODO: comment and code don't match - weekend?!!
-    # 3 bits for season, 1 bit for day of week, 2 for weekend, 5 for time of day
```

Comment:
@scottpurdy Any idea the history of this comment?  IT seems neither the comment, nor the call to constructor are correct.  According to docstring, it should only be 0 or 1, although technically it could be any odd number since it becomes `w` for the `ScalarEncoder`.